website/pages/blog/auditing-service-account-keys-with-cloudquery.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Auditing Service Account Keys with CloudQuery
+tag: security
+date: 2022/11/04
+description: >-
+  A case against Service Account Keys.
+author: SCKelemen, Alex
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+
+# Intro
+There are several common methods of accessing cloud 
+environments, each with their own risks and uses. Service 
+Account Keys, however, are one of the more dangerous methods.
+Before using service account keys, there are several risks 
+and shortfalls you should consider:
+
+Service Account Keys offer a simple and easy mechanism for 
+granting access to cloud resources. Service Account Keys are 
+easy to understand, and are compatible with most existing 
+architectures and systems. 
+
+Initially, you might be appealed by the ease of using 
+account keys. All you need to do is simply put the key
+somewhere your systems can access. This could be a file,
+environment, variable, or bucket.
+
+Unfortunately, with this highly flexible global bearer strategy,
+tracking the location of the key becomes unmanageable. Multiple 
```

Comment:
For your consideration: this could be characterized as usage sprawl - where multiple users and services can share usage of the same key.

```suggestion
tracking the location of the key becomes unmanageable. Multiple 
```